DataFormats/L1TrackTrigger/interface/TTStub.h

```diff
@@ -31,31 +31,77 @@ class TTStub {
   /// Data members:   getABC( ... )
   /// Helper methods: findABC( ... )
 
-  /// Clusters composing the Stub
-  const edm::Ref<edmNew::DetSetVector<TTCluster<T> >, TTCluster<T> >& getClusterRef(unsigned int hitIdentifier) const;
+  /// Clusters composing the Stub -- see https://twiki.cern.ch/twiki/bin/viewauth/CMS/SLHCTrackerTriggerSWTools#TTCluster
+
+  /// Returns the permanent references of the cluster in the sensor stack identified by hitStackMember
+  /// which should be either 0 or 1 for the innermost and outermost sensor, respectively
+  const edm::Ref<edmNew::DetSetVector<TTCluster<T> >, TTCluster<T> >& getClusterRef(unsigned int hitStackMember) const;
+
+  /// Add a cluster reference, depending on which stack member it is on (inner = 0, outer = 1)
   void addClusterRef(edm::Ref<edmNew::DetSetVector<TTCluster<T> >, TTCluster<T> > aTTCluster);
 
   /// Detector element
   DetId getDetId() const { return theDetId; }
   void setDetId(DetId aDetId) { theDetId = aDetId; }
 
-  /// Trigger information
-  double getTriggerDisplacement() const;           /// In FULL-STRIP units! (hence, not implemented herein)
-  void setTriggerDisplacement(int aDisplacement);  /// In HALF-STRIP units!
-  double getTriggerOffset() const;                 /// In FULL-STRIP units! (hence, not implemented herein)
-  void setTriggerOffset(int anOffset);             /// In HALF-STRIP units!
-  double getRealTriggerOffset() const;             /// In FULL-STRIP units! (hence, not implemented herein)
-  void setRealTriggerOffset(float anOffset);       /// In HALF-STRIP units!
+  /// Trigger information; see, e.g., TTStubAlgorithm_official::PatternHitCorrelation for definitions
+  /// Values are passed back from there via the TTStubBuilder, be careful to choose the right one.
+  /// In particular note the difference between values passed as "full" strip units (i.e. the strip
+  /// number or difference between strip numbers) and "half" strip units, which have a 2X finer granularity.
+
+  /// getRawBend() [rename of getTriggerDisplacement()]: In FULL strip units!
+  /// Returns the relative displacement between the two cluster centroids, i.e.
+  /// the difference between average row coordinates in inner and outer stack member,
+  /// in terms of outer member pitch (if both pitches are the same, this is just the coordinate difference).
+  /// Flag for rejected stubs: +500 if rejected by FE, +1000 if rejected by CIC chip.
+  double getRawBend() const;
```

Comment:
usual comment that the [CMS Coding Rules](http://cms-sw.github.io/cms_coding_rules.html) recommend against using "get" for getters